### PR TITLE
Fix Hover

### LINKED ISSUE #8717: [UI]Meshmap Icon doesn't move on hovering.
### Current Behavior
On the sidebar of Meshery when collapsed Meshmap Icon doesn't move on hover.

### Desired Behavior
The Meshmap Icon should move on hover similar to other icons.

### Screenshots/Logs
https://github.com/meshery/meshery/assets/90668597/ce6c3c23-62e7-499b-b7e5-f73ba2f75d8e

### Environment
- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ptdatta**
**Notes for Reviewers**

This PR fixes #8717 

[Screencast from 02-10-23 01:34:02 AM IST.webm](https://github.com/meshery/meshery/assets/94271200/04885bcf-d908-425e-af21-f0d4beda48d2)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Navigator.js [hunk 1]
```diff
@@ -610,7 +610,20 @@ class Navigator extends React.Component {
...
+                e.target.style.transform = 'translate(-20%, -25%)';
+                e.target.style.top = '0';
+                e.target.style.right = '0';
+              }}
+              onMouseOut={(e) => {
+                e.target.style.transform = 'translate(0, 0)';
+                e.target.style.top = 'auto';
+                e.target.style.right = 'auto';
+              }}
+            />
```
Comment (theBeginner86) [inline ui/components/Navigator.js:626]: This is good, but can we use standardizing through the use of out themes? // @Modern-artist @ritiksaxena124

Comment (ptdatta) [reply ui/components/Navigator.js:626]: I was trying to do that. But I don't know why it doesn't work. One potential reason can be -
  - The other icons of the navigator were coming natively but the meshmap icon was coming from the web. The other icons are JS versions of SVG but meshmap icon is pure SVG so the CSS wasn't working.

**Comment by leecalcote** [comment]
The resultant effect looks good. Please do consider incorporating, @theBeginner86's feedback

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by ritiksaxena124** [comment]
LGTM 👍now

**Review by leecalcote — APPROVED** [review]

**Pull request merged**